docs/start/install.rst

```diff
@@ -66,44 +70,76 @@ The following platforms are supported:
 officially released, then you can use one of the binary wheels that we build
-as part of our Continuous Integration process.
+as part of our Continuous Integration process. 
 
-If you are on Windows, then pre-built wheels are available on `AppVeyor`_.
-Click on a green main build of your choice, then navigate to the "Artifacts"
-tab, copy the wheel URL that corresponds to your Python version, and finally
-install it as:
+There are two sources available:
 
-.. code-block:: doscon
+- For macOS and Linux please use our `S3 repository`_. 
+
+- For Windows installations, you can find the latest wheels within our
+  `AppVeyor`_ build system.
+
+
+Install from the Amazon S3 Repository (macOS & Linux)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+First it is **recommended** to :ref:`Create a Python Virtual Environment<PVE Linux & macOS>`.
+
+Both the most current and historical development wheels for macOS & Linux can 
+be found at our `S3 repository`_.
 
-    C:\> pip install YOUR_WHEEL_URL
+- To install the most current development version, **scroll to the bottom** of 
+  the page to find the latest links, and then download or copy the URL of a
+  wheel that corresponds to your Python version and platform. 
 
-For macOS and Linux, development wheels can be found at our `S3 repository`_.
-Scroll to the bottom of the page to find the latest links, and then download
-or copy the URL of a wheel that corresponds to your Python version and
-platform. This wheel can be installed with ``pip`` as usual:
+  .. image:: ../_static/amazon_s3_repo.png
+    :scale: 25 %
+    :alt: datatable S3 binary wheel links
+
+- This wheel can be installed with ``pip`` as usual:
 
 .. code-block:: console
 
-    $ pip install YOUR_WHEEL_URL
+    $ pip install https://AWS S3 DATATABLE BUILD WHEEL URL.whl
 
+Install from AppVeyor (Windows)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+First it is **recommended** to :ref:`Create a Python Virtual Environment<PVE Windows>`.
 
-Build from source
------------------
+- To find the latest build wheels for datatable, go to datatable's `AppVeyor`_ 
+  CI/CD Instance and select the platform you're installing for:
 
-In order to build and install the latest development version of ``datatable``
-directly from GitHub, run the following command:
+  .. image:: ../_static/appveyor-platform.png
+    :scale: 25 %
+    :alt: datatable appveyor platform links
+
+- next, select **"Artifacts"** and **right/option-click** on the filename for 
+  your desired build wheel.
+- then, Select "Copy Link Address" from your browser's context menu to copy 
+  the wheel's URL
+
+  .. image:: ../_static/appveyor-artifacts.png
+    :scale: 25 %
+    :alt: datatable appveyor artifact links
+
+- Finally, install using :code:`pip install` by pasting the copied url.
+
+.. code-block:: doscon
+
+    C:\> pip install https://APPVEYOR DATATABLE BUILD WHEEL URL.whl
 
-.. code-block:: console
 
-   $ pip install git+https://github.com/h2oai/datatable
```

Comment:
Since this is a common part for macOS / Linux / Windows, why to remove it? May be just leave it, then call the following sections "Installing/configuring C++ compiler" for various platforms?